code/datums/extensions/movement/rotate_facing.dm

```diff
@@ -47,7 +47,9 @@
 
 	target = newtarget
 	GLOB.moved_event.register(target, src, target_moved())
-	start_tracking()
```

Comment:
this is a step backwards, please revert changes to this file.

Wrapper functions like this are important for future extensibility, and they're useful right now if there's more than one line of code